src/libopensc/card-tcos.c

```diff
@@ -722,7 +723,6 @@ struct sc_card_driver * sc_get_tcos_driver(void)
 	tcos_ops.select_file          = tcos_select_file;
 	tcos_ops.list_files           = tcos_list_files;
 	tcos_ops.delete_file          = tcos_delete_file;
-	tcos_ops.set_security_env     = tcos_set_security_env;
```

Comment:
did you delete the reference to `tcos_set_security_env` by accident? It is not used elsewhere...

Reply:
I just saw that it's a duplicate, nevermind